charts/tfy-clickhouse/values.yaml

```diff
@@ -27,6 +27,10 @@ altinity-clickhouse-operator:
     enabled: true
     additionalLabels:
       release: prometheus
+  serviceAccount:
+    create: true
+    name: tfy-clickhouse-operator
```

Comment:
By default a serviceaccount is getting created with the name `truefoundry-altinity-clickhouse-operator`. We don't need to specify one from our side. Also automount should be default to `true` as k8s uses [opt-out](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#opt-out-of-api-credential-automounting) strategy